.gitignore

```diff
@@ -4,3 +4,4 @@ node_modules
 # Twelvety caches responsive images
 # Don't commit otherwise your build won't generate any images
 .twelvety.cache
+yarn.lock
```

Comment:
I'm not sure if we want to ignore `yarn.lock`, in case someone wants to switch from npm to Yarn. What do you think?

Reply:
You don't have to include it; I couldn't remove it from my commit and assumed you would ignore it during the PR if you didn't want.   From a technical side, you can ignore package.lock and yarn.lock as both are dynamically created on install and don't need to be part of any repository.

Reply:
That's alright, I've removed it from this pull request but users can add it locally if they want. It's usually best practice to commit lock files https://classic.yarnpkg.com/blog/2016/11/24/lockfiles-for-all/

Reply:
Thanks for the article!